content/project-structure/src/fonts.md

```diff
@@ -2,29 +2,91 @@
 title: The Fonts folder
 ---
 
-Custom fonts are placed in the `src/fonts` folder. This folder is treated special, because `.ttf` files placed here are automatically picked up by NativeScript.
+A dedicated folder for storing custom fonts. Any `.ttf` files placed here are automatically picked up by NativeScript.
 
 ## Using Custom Fonts
 
-<!-- TODO: use concrete font as an example, and show ns fonts output -->
-
-To use a custom font, place the `.ttf` file in the `src/fonts` folder, then run
+Place the `.ttf` file(s) in the `src/fonts` folder, then run
 
 ```cli
 ns fonts
 ```
 
-to generate the necessary css declarations. You can manually write these, the above command is just a convenience method to save some time and reduce the guesswork, since the `font-family` can be different from the file name.
+The result is a table with the font file name and the necessary CSS properties you can copy/paste into your css:
+
+| Font                  | CSS Properties                                                            |
+| --------------------- | ------------------------------------------------------------------------- |
+| `albert-sans-700.ttf` | `font-family: "Albert Sans", "albert-sans-700"; font-weight: 700;`        |
+| `fa-regular-400.ttf`  | `font-family: "Font Awesome 5 Free", "fa-regular-400"; font-weight: 400;` |
 
-Create a new css class in your CSS, for example in [app.css or app.scss](/project-structure/src/app-css-scss)
+Add a new css class (for example in [`app.css`](/project-structure/src/app-css-scss)) with the **CSS Properties** from the table (the name of the class can be anything)
 
 ```css
-.custom-font-class {
-  font-family: '...';
+.font-albert-sans {
+  font-family: 'Albert Sans', 'albert-sans-700';
+  font-weight: 700;
+}
+
+.far {
+  font-family: 'Font Awesome 5 Free', 'fa-regular-400';
   font-weight: 400;
 }
 ```
 
-## Using Icon Fonts
+::: info Note
+Using `ns fonts` is optional, however we recommend using it because there is a conceptual difference in how **.ttf** fonts are recognized on iOS and Android. On Android, the font is recognized by its **file name** while on iOS it is recognized by its **font name**. This means that fonts that are created with a font name which is different from the file name has to be registered with both names in your CSS rule.
```

Comment:
```suggestion
Using `ns fonts` is optional, however we recommend using it because iOS and Android recognize `.ttf` files differently. On Android, the font is recognized by its **file name** while on iOS it is recognized by its **font name**. Fonts that have a different font name than the file name have to be registered with both names in the CSS rule.
```